tests/codestyle/CodestyleTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+use Composer\Console\Application;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+
+class CodestyleTest extends TestCase
+{
+    protected const FIXTURES = [
+        "singleQuotes",
+        "strictTypes",
+        "unionTypes",
+    ];
```

Comment:
I'm not sure if that's such a great idea, but instead of explicitly specifying the fixtures here, we could just read the contents of `codestyle/fixtures` folder. This way adding a new test case will just involve creating new fixture dir.
On the other hand this would be a little less readable so maybe not worth it.